### PR TITLE
Update version to latest releases

### PR DESCRIPTION
**Pull request by navidshaikh**
0.19.1
 0.18.4
 0.17.4

### CODE REVIEW

File: kn@0.17.rb [hunk 1]
```diff
@@ -12,10 +12,10 @@ class KnAT017 < Formula
 
   if OS.mac?
     url "https://github.com/knative/client/releases/download/#{v}/kn-darwin-amd64"
-    sha256 "8f7b9f2dd11f2bcabdc89c50a4b865e7822cf1e815131b35435f2335199ee0d9"
+    sha256 "63c8cec08d4f5d9ea4c960c063813a679ccdca73409942b18c1a427d8ccf085e"
```
Comment (rhuss) [inline kn@0.17.rb:15]: please update the version to 0.17.4, too, in this file.

Comment (navidshaikh) [reply kn@0.17.rb:15]: fixed

**Review by maximilien — APPROVED** [review]
/lgtm

**Pull request merged**